Index: marble/data/CMakeLists.txt
===================================================================
--- marble.orig/data/CMakeLists.txt	2015-06-12 17:16:29.442375420 +0200
+++ marble/data/CMakeLists.txt	2015-06-12 17:19:21.483536302 +0200
@@ -101,7 +101,6 @@
 DESTINATION ${MARBLE_DATA_INSTALL_PATH}/maps/earth/plain)
 
 install (FILES
-maps/earth/openstreetmap/openstreetmap.dgml
 maps/earth/openstreetmap/openstreetmap-preview.png
 maps/earth/openstreetmap/legend.html
 maps/earth/openstreetmap/tour.kml
@@ -152,7 +151,6 @@
 install (FILES
 maps/earth/temp-july/temp-july.jpg
 maps/earth/temp-july/temp-july-preview.png
-maps/earth/temp-july/temp-july.dgml
 DESTINATION ${MARBLE_DATA_INSTALL_PATH}/maps/earth/temp-july)
 
 install (FILES
