commit 6b1117db6b0721cf1fac980273068925858028de
Author: Torsten Rahn <rahn@kde.org>
Date:   Sat Oct 10 13:38:47 2015 +0200

    Clarify license of the DGML files.

Index: marble/data/maps/earth/bluemarble/bluemarble.dgml
===================================================================
--- marble.orig/data/maps/earth/bluemarble/bluemarble.dgml	2015-10-12 17:37:14.435768830 +0200
+++ marble/data/maps/earth/bluemarble/bluemarble.dgml	2015-10-12 17:37:14.431768990 +0200
@@ -1,7 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
+     This xml file is free software licensed under the GNU LGPL. You can
+     find a copy of this license in LICENSE.txt in the top directory of
+     the source code.
+-->
 <dgml xmlns="http://edu.kde.org/marble/dgml/2.0">
     <document>
         <head>
+            <!-- the license here applies to the actual map content, not this dgml file. See the license header above for the dgml file license -->
             <license short="NASA / Public Domain">Source: NASA Goddard Space Flight Center Earth Observatory http://earthobservatory.nasa.gov/Newsroom/BlueMarble, License: Public Domain</license>
             <name>Satellite View</name>
             <target>earth</target>
Index: marble/data/maps/earth/citylights/citylights.dgml
===================================================================
--- marble.orig/data/maps/earth/citylights/citylights.dgml	2015-10-12 17:37:14.435768830 +0200
+++ marble/data/maps/earth/citylights/citylights.dgml	2015-10-12 17:37:14.431768990 +0200
@@ -1,7 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
+     This xml file is free software licensed under the GNU LGPL. You can
+     find a copy of this license in LICENSE.txt in the top directory of
+     the source code.
+-->
 <dgml xmlns="http://edu.kde.org/marble/dgml/2.0">
     <document>
         <head>
+            <!-- The license here applies to the actual map content, not this dgml file. See the license header above for the dgml file license -->
             <license short="NOAA NGDC, NASA / Public Domain">Source: Marc Imhoff of NASA GSFC; Christopher Elvidge of NOAA NGDC; Craig Mayhew and Robert Simmon, NASA GSFC, License: Public Domain</license>
             <name>Earth at Night</name>
             <target>earth</target>
Index: marble/data/maps/earth/openstreetmap/openstreetmap.dgml
===================================================================
--- marble.orig/data/maps/earth/openstreetmap/openstreetmap.dgml	2015-10-12 17:37:14.435768830 +0200
+++ marble/data/maps/earth/openstreetmap/openstreetmap.dgml	2015-10-12 17:37:14.431768990 +0200
@@ -1,7 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
+     This xml file is free software licensed under the GNU LGPL. You can
+     find a copy of this license in LICENSE.txt in the top directory of
+     the source code.
+-->
 <dgml xmlns="http://edu.kde.org/marble/dgml/2.0">
   <document>
     <head>
+      <!-- The license here applies to the actual map content, not this dgml file. See the license header above for the dgml file license -->
+      <!-- The content that this license refers to is not distributed with this package -->
       <license short="© OpenStreetMap contributors">Source: © OpenStreetMap contributors, License: Creative Commons Attribution-ShareAlike 2.0 (CC BY-SA)</license>
       <name>OpenStreetMap</name>
       <target>earth</target>
Index: marble/data/maps/earth/plain/plain.dgml
===================================================================
--- marble.orig/data/maps/earth/plain/plain.dgml	2015-10-12 17:37:14.435768830 +0200
+++ marble/data/maps/earth/plain/plain.dgml	2015-10-12 17:37:14.431768990 +0200
@@ -1,8 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
+     This xml file is free software licensed under the GNU LGPL. You can
+     find a copy of this license in LICENSE.txt in the top directory of
+     the source code.
+-->
 <dgml xmlns="http://edu.kde.org/marble/dgml/2.0">
     <document>
         <head>
-            <license short="CIA / Public Domain" attribution="always">Source: Micro World Data Bank in Polygons („MWDB-POLY / MWDBII“), CIA; Global Associates, Ltd.; Fred Pospeschil und Antonio Rivera, License: Public Domain</license>
+            <!-- the license here applies to the actual map content, not this dgml file. See the license header above for the dgml file license -->
+            <license short="Public Domain">Source: Made with Natural Earth. Free vector and raster map data @ naturalearthdata.com., License: Public Domain</license>
             <name>Plain Map</name>
             <target>earth</target>
             <theme>plain</theme>
Index: marble/data/maps/earth/political/political.dgml
===================================================================
--- marble.orig/data/maps/earth/political/political.dgml	2015-10-12 17:37:14.435768830 +0200
+++ marble/data/maps/earth/political/political.dgml	2015-10-12 17:37:14.431768990 +0200
@@ -1,7 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
+     This xml file is free software licensed under the GNU LGPL. You can
+     find a copy of this license in LICENSE.txt in the top directory of
+     the source code.
+-->
 <dgml xmlns="http://edu.kde.org/marble/dgml/2.0">
     <document>
         <head>
+            <!-- the license here applies to the actual map content, not this dgml file. See the license header above for the dgml file license -->
+            <license short="Public Domain">Source: Made with Natural Earth. Free vector and raster map data @ naturalearthdata.com., License: Public Domain</license>
             <name>Political Map</name>
             <target>earth</target>
             <theme>political</theme>
Index: marble/data/maps/earth/precip-dec/precip-dec.dgml
===================================================================
--- marble.orig/data/maps/earth/precip-dec/precip-dec.dgml	2015-10-12 17:37:14.435768830 +0200
+++ marble/data/maps/earth/precip-dec/precip-dec.dgml	2015-10-12 17:37:14.431768990 +0200
@@ -1,7 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
+     This xml file is free software licensed under the GNU LGPL. You can
+     find a copy of this license in LICENSE.txt in the top directory of
+     the source code.
+-->
 <dgml xmlns="http://edu.kde.org/marble/dgml/2.0">
     <document>
         <head>
+            <!-- the license here applies to the actual map content, not this dgml file. See the license header above for the dgml file license -->
+            <license short="Legates, Willmott / CC by-SA 3.0">Legates, D.R. und Willmott, C.J. 1989. Average Monthly Surface Air Temperature and Precipitation</license>
             <name>Precipitation (December)</name>
             <target>earth</target>
             <theme>precip-dec</theme>
Index: marble/data/maps/earth/precip-july/precip-july.dgml
===================================================================
--- marble.orig/data/maps/earth/precip-july/precip-july.dgml	2015-10-12 17:37:14.435768830 +0200
+++ marble/data/maps/earth/precip-july/precip-july.dgml	2015-10-12 17:37:14.431768990 +0200
@@ -1,7 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
+     This xml file is free software licensed under the GNU LGPL. You can
+     find a copy of this license in LICENSE.txt in the top directory of
+     the source code.
+-->
 <dgml xmlns="http://edu.kde.org/marble/dgml/2.0">
     <document>
         <head>
+            <!-- the license here applies to the actual map content, not this dgml file. See the license header above for the dgml file license -->
+            <license short="Legates, Willmott / CC by-SA 3.0">Legates, D.R. und Willmott, C.J. 1989. Average Monthly Surface Air Temperature and Precipitation</license>
             <name>Precipitation (July)</name>
             <target>earth</target>
             <theme>precip-july</theme>
Index: marble/data/maps/earth/schagen1689/schagen1689.dgml
===================================================================
--- marble.orig/data/maps/earth/schagen1689/schagen1689.dgml	2015-10-12 17:37:14.435768830 +0200
+++ marble/data/maps/earth/schagen1689/schagen1689.dgml	2015-10-12 17:37:14.431768990 +0200
@@ -1,7 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
+     This xml file is free software licensed under the GNU LGPL. You can
+     find a copy of this license in LICENSE.txt in the top directory of
+     the source code.
+-->
 <dgml xmlns="http://edu.kde.org/marble/dgml/2.0">
     <document>
         <head>
+            <!-- the license here applies to the actual map content, not this dgml file. See the license header above for the dgml file license -->
+            <license short="Public Domain">Source: Wikipedia, License: Public Domain</license>
             <name>Historical Map 1689</name>
             <target>earth</target>
             <theme>schagen1689</theme>
Index: marble/data/maps/earth/srtm/srtm.dgml
===================================================================
--- marble.orig/data/maps/earth/srtm/srtm.dgml	2015-10-12 17:37:14.435768830 +0200
+++ marble/data/maps/earth/srtm/srtm.dgml	2015-10-12 17:37:14.431768990 +0200
@@ -1,7 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
+     This xml file is free software licensed under the GNU LGPL. You can
+     find a copy of this license in LICENSE.txt in the top directory of
+     the source code.
+-->
 <dgml xmlns="http://edu.kde.org/marble/dgml/2.0">
     <document>
         <head>
+            <!-- the license here applies to the actual map content, not this dgml file. See the license header above for the dgml file license -->
+            <license short="Public Domain">Source: Made with Natural Earth. Free vector and raster map data @ naturalearthdata.com., License: Public Domain</license>
             <name>Atlas</name>
             <target>earth</target>
             <theme>srtm</theme>
Index: marble/data/maps/earth/srtm2/srtm2.dgml
===================================================================
--- marble.orig/data/maps/earth/srtm2/srtm2.dgml	2015-10-12 17:37:14.435768830 +0200
+++ marble/data/maps/earth/srtm2/srtm2.dgml	2015-10-12 17:37:14.431768990 +0200
@@ -1,4 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
+     This xml file is free software licensed under the GNU LGPL. You can
+     find a copy of this license in LICENSE.txt in the top directory of
+     the source code.
+-->
 <dgml xmlns="http://edu.kde.org/marble/dgml/2.0">
     <document>
         <head>
Index: marble/data/maps/earth/temp-dec/temp-dec.dgml
===================================================================
--- marble.orig/data/maps/earth/temp-dec/temp-dec.dgml	2015-10-12 17:37:14.435768830 +0200
+++ marble/data/maps/earth/temp-dec/temp-dec.dgml	2015-10-12 17:37:14.431768990 +0200
@@ -1,7 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
+     This xml file is free software licensed under the GNU LGPL. You can
+     find a copy of this license in LICENSE.txt in the top directory of
+     the source code.
+-->
 <dgml xmlns="http://edu.kde.org/marble/dgml/2.0">
     <document>
         <head>
+            <!-- the license here applies to the actual map content, not this dgml file. See the license header above for the dgml file license -->
+            <license short="Legates, Willmott / CC by-SA 3.0">Legates, D.R. und Willmott, C.J. 1989. Average Monthly Surface Air Temperature and Precipitation</license>
             <name>Temperature (December)</name>
             <target>earth</target>
             <theme>temp-dec</theme>
Index: marble/data/maps/earth/temp-july/temp-july.dgml
===================================================================
--- marble.orig/data/maps/earth/temp-july/temp-july.dgml	2015-10-12 17:37:14.435768830 +0200
+++ marble/data/maps/earth/temp-july/temp-july.dgml	2015-10-12 17:39:03.415418566 +0200
@@ -1,8 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
+     This xml file is free software licensed under the GNU LGPL. You can
+     find a copy of this license in LICENSE.txt in the top directory of
+     the source code.
+-->
 <dgml xmlns="http://edu.kde.org/marble/dgml/2.0">
     <document>
         <head>
-            <license short="Legates, Willmott / CC by-SA">Legates, D.R. und Willmott, C.J. 1989. Average Monthly Surface Air Temperature and Precipitation, License: Creative Commons Attribution Share-Alike 2.0 (CC BY-SA)</license>
+            <!-- the license here applies to the actual map content, not this dgml file. See the license header above for the dgml file license -->
+            <license short="Legates, Willmott / CC by-SA 3.0">Legates, D.R. und Willmott, C.J. 1989. Average Monthly Surface Air Temperature and Precipitation</license>
             <name>Temperature (July)</name>
             <target>earth</target>
             <theme>temp-july</theme>
Index: marble/data/maps/sky/nightsky/nightsky.dgml
===================================================================
--- marble.orig/data/maps/sky/nightsky/nightsky.dgml	2015-10-12 17:37:14.435768830 +0200
+++ marble/data/maps/sky/nightsky/nightsky.dgml	2015-10-12 17:37:14.435768830 +0200
@@ -1,8 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
+     This xml file is free software licensed under the GNU LGPL. You can
+     find a copy of this license in LICENSE.txt in the top directory of
+     the source code.
+-->
 <dgml xmlns="http://edu.kde.org/marble/dgml/2.0">
     <document>
         <head>
-            <license short="CIA / Public Domain" attribution="always">Source: Yale Catalogue of Bright Stars, License: Public Domain</license>
+            <license short="Public Domain" attribution="always">Source: Yale Catalogue of Bright Stars, License: Public Domain</license>
             <name>Night Sky</name>
             <target>sky</target>
             <theme>nightsky</theme>
