Index: marble-15.08.1/cmake/modules/Findquazip.cmake
===================================================================
--- marble-15.08.1.orig/cmake/modules/Findquazip.cmake
+++ marble-15.08.1/cmake/modules/Findquazip.cmake
@@ -7,8 +7,8 @@ if(QUAZIP_INCLUDE_DIR AND QUAZIP_LIBRARI
 else(QUAZIP_INCLUDE_DIR AND QUAZIP_LIBRARIES)
 
   if(QT5BUILD)
-  find_path(QUAZIP_INCLUDE_DIR NAMES quazip.h PATH_SUFFIXES quazip5)
-  find_library(QUAZIP_LIBRARIES NAMES quazip5)
+  find_path(QUAZIP_INCLUDE_DIR NAMES quazip.h PATH_SUFFIXES quazip5 quazip)
+  find_library(QUAZIP_LIBRARIES NAMES quazip5 quazip-qt5)
   else(QT5BUILD)
   find_path(QUAZIP_INCLUDE_DIR NAMES quazip.h PATH_SUFFIXES quazip)
   find_library(QUAZIP_LIBRARIES NAMES quazip)
